types/hapi__nes/index.d.ts

```diff
@@ -42,7 +42,7 @@ declare module '@hapi/hapi' {
     }
 }
 
-declare module nes {
+declare namespace nes {
```

Comment:
Looks like this module has a compile error in TypeScript 4.8 but not 4.9, unrelated to this change. https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/5767649258/job/15637662697?pr=66292

<details>
<summary>Errors copied from CI</summary>

```plaintext
Error in hapi__nes
Error: /home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/broadcast-server.ts:6:20
ERROR: 6:20   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.

/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/hapi__nes-tests.ts:4:20
ERROR: 4:20   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.
ERROR: 23:15  expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
Parameter 'err' implicitly has an 'any' type.

/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/route-authentication-server.ts:8:20
ERROR: 8:20   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.

/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/route-invocation-server.ts:6:20
ERROR: 6:20   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.

/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/subscription-filter-server.ts:8:18
ERROR: 8:18   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.
ERROR: 48:33  expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
Parameter 'options' implicitly has an 'any' type.

/home/runner/work/DefinitelyTyped/DefinitelyTyped/types/hapi__nes/test/subscriptions-server.ts:6:18
ERROR: 6:18   expect  Compile error in typescript@4.8 but not in typescript@4.9.
Fix with a comment '// Minimum TypeScript Version: 4.9' just under the header.
'Server' only refers to a type, but is being used as a value here.
```

</details>

Reply:
this one is not very active (native module), the source on DT can be removed (to skip investigation and working on fixing the types), I could do this in parallel

Reply:
PR: let's give the owners a chance to make an opinion:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66372

Reply:
amended in master branch

Reply:
Thanks! Merged from `master`. 👍